docs/core/entity/vacuum.md

```diff
@@ -40,6 +40,8 @@ Properties should always only return information from memory and not do I/O (lik
 
 Supported features are defined by using values in the `VacuumEntityFeature` enum
 and are combined using the bitwise or (`|`) operator.
+Note that all vacuum entity platforms derived from `homeassistant.components.vacuum.StateVacuumEntity`
```

Comment:
I don't think this should be a supported feature of its required.

Reply:
I agree, but this PR documents how to implement a vacuum platform given the current core implementation.

Reply:
As discussed separately, this feature is currently needed for the frontend to identify the new state vacuum from the legacy vacuum entity.

We'll plan to remove it when the legacy entity is removed.